editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/pages/FuseIntegrationProjectWizardRuntimeAndCamelPage.java

```diff
@@ -293,7 +310,7 @@ private void configureRuntimeCombo() throws CoreException {
 			runtimesSet = RuntimeManager.getRuntimes(Collections.singleton(facetVersion));
 		}
 		
-		Map<String, IRuntime> runtimesMap = new LinkedHashMap<>();
+		Map<String, IRuntime> runtimesMap = new LinkedHashMap<String,IRuntime>();
```

Comment:
optional:  why did you add the generic types? It is simpler to stick with diamond operator new LinkedHashMap<>()